docs/cookbook/agents/sql_agent.md

```diff
@@ -164,14 +162,11 @@ class Librarian(BaseModel):
         You must use these tools to interact with the database.
 
         MESSAGES: {self.messages}
-        USER:
-        {query}
+        USER: {query}
         """
     )
     async def _step(self, query: str) -> openai.OpenAIDynamicConfig:
-        return {
-            "tools": [self._run_query, self._execute_query],
-        }
+        return {"tools": [self._run_query, self._execute_query]}
         
     async def _get_response(self, question: str):
```

Comment:
maybe `_get_response` is not the right naming for this function since it doesn't return anything.

I think a better structure would be `run` calls `_step` that calls `_call`